website/spec/v0.1/did-plc.md

```diff
@@ -222,13 +222,62 @@ To fully validate a DID document against the operation log:
     - if no key matches, there has been a trust violation; the PLC server should never have accepted the operation
 - verify the correctness of "nullified" operations and the current active operation log using the rules around rotation keys and recovery windows
 
-The complete log of operations for all DIDs on the PLC server can be enumerated efficiently:
 
-- HTTP endpoint: `https://plc.directory/export`
-- output format: [JSON lines](https://jsonlines.org/)
-- `count` query parameter, as an integer, maximum 1000 lines per request
-- `after` query parameter, based on `createdAt` timestamp, for pagination
+### Bulk Export
 
+The complete log of operations for all DIDs on the PLC server can be enumerated efficiently.
+
+The HTTP endpoint `https://plc.directory/export` returns data in [JSON lines](https://jsonlines.org/) format. It supports two query parameters:
+
+- `count`: An integer, maximum 1000 lines per request.
+- `after`: The cursor for pagination, either an integer or a timestamp string (see below).
+
+When `after` is an integer, each JSON line has the following fields:
+
+- `type` (string): A string with fixed value `indexed_op`.
+- `operation` (object): A valid PLC create or update operation.
+- `did` (string): The corresponding `did:plc:` identifier.
+- `cid` (string): The computed CID of the `operation` value.
+- `createdAt` (string): The timestamp corresponding to when the operation was accepted by the directory, in `YYYY-MM-DDTHH:mm:ss.sssZ` format.
+- `seq` (integer): A sequence number. The returned entries have strictly monotonic sequence numbers. (See also: "Sequence Number Semantics")
+
+The `seq` value of the final JSON line can be used as the `after` cursor value to retrieve the next page of results.
+
+Alternatively, when `after` is a timestamp string (in `YYYY-MM-DDTHH:mm:ss.sssZ` format), each JSON line has the following fields:
```

Comment:
I would more aggressively call out that this is a legacy format. Encourage folks not to use it, and say it may be deprecated at some point.